src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -1,5 +1,8 @@
 (ns status-im.contexts.wallet.send.utils
```

Comment:
what's the difference between wallet/common/utils/send and this one? 

I see that the files can have different uses, but is there a pattern we are following?

Reply:
The difference lies in their scope of use: wallet/common/utils/send likely contains utility functions used broadly across the wallet application, while the other is specialized for use within the sibling namespaces (in this case in `events.cljs`). Open to move it of course!

Reply:
Nope that sounds perfect! 🙌🏼